fuel-indexer-tests/tests/e2e.rs

```diff
@@ -145,3 +145,79 @@ async fn test_can_trigger_and_index_log_event() {
 //     // TODO: finish
 //     assert_eq!(1, 1);
 // }
+
+// TODO: Need to find a better way to test ScriptResult receipt indexing as
+// the expected values for this test will be overwritten by any test
+// with an event that involves a ScriptResult receipt, which may
+// make the assertions fail as tests are not called in a particular order;
+// one can verify that indexing works by running this test by itself
+
+// #[tokio::test]
```

Comment:
@deekerno 
- I see the `TODO` but we should still have a test for `ScriptResult` (since that's actually the easiest to generate)
- Not a blocker to the PR getting merged -- but as a very last resort, to test this in isolation, we could just remove all other `ScriptResult`s in this test before calling `client.post()`
  - We could even just have this test `sqlx::query` a previous test's `ScriptResult`